content/contact/index.md

```diff
@@ -0,0 +1,9 @@
+# Contact
+
+If you're on [Slack](/slack), you can directly message one of the [Leadership](/leadership) team.
+
+For website comments, see [GitHub](https://github.com/f3peakcity/f3peakcity.github.io).
+
+If you're new to F3, visiting from out of town, or just have a general question or comment, please fill out the following form.
+
+<iframe src="https://docs.google.com/forms/d/e/1FAIpQLSeVPJAdsFzyyyXDqA1WTic_62V291rdESRlMsvUJfKtt1zdWg/viewform?embedded=true" width="640" height="752" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
```

Comment:
Embedded a Google Form to make sure it works...would like to transfer to the Peak City gmail and get it setup to forward there.

Reply:
amended your commit and switched the form link with one hosted by peak city gmail. Thanks!